packages/webservice/app/src/main/resources/db/changelog/changelog.xml

```diff
@@ -84,28 +84,55 @@
              FOREIGN KEY (user_id) references User(id)
          );
 
-         CREATE TABLE map_user_to_fav_reports (
+          CREATE TABLE map_user_to_fav_reports (
              user_id varchar(255) not null,
              report_id integer not null,
              PRIMARY KEY (user_id, report_id),
              FOREIGN KEY (report_id) references custom_reports(id),
              FOREIGN KEY (user_id) references User(id)
-         );
+          );
 
-         CREATE TABLE Role (
+          CREATE TABLE Role (
              id varchar(255) not null,
              createdOn timestamp default current_timestamp,
              updatedOn timestamp default current_timestamp,
              PRIMARY KEY (id)
-        );
+          );
 
-        CREATE TABLE User_Roles (
-            user_id varchar(255) not null,
-            roles_id varchar(255) not null,
-            PRIMARY KEY (user_id, roles_id),
-            FOREIGN KEY (roles_id) references Role(id),
-            FOREIGN KEY (user_id) references User(id)
-        );
+          CREATE TABLE User_Roles (
+             user_id varchar(255) not null,
+             roles_id varchar(255) not null,
+             PRIMARY KEY (user_id, roles_id),
+             FOREIGN KEY (roles_id) references Role(id),
+             FOREIGN KEY (user_id) references User(id)
+          );
+        </sql>
+    </changeSet>
+    <changeSet author="admin" id="2">
+        <sql>
+          ALTER TABLE AsyncQuery
+          ADD FOREIGN KEY (principalName) references User(id);
+
+          CREATE TABLE TableExport (
+            id varchar(36) not null,
+            createdOn timestamp,
+            principalName varchar(255),
+            query varchar(255),
+            queryType integer,
+            requestId varchar(36),
+            resultType varchar(255),
+            status varchar(255),
+            updatedOn timestamp,
+            contentLength integer,
+            responseBody text,
+            completedOn timestamp,
+            httpStatus integer,
+            recordCount integer,
+            message text,
```

Comment:
```suggestion
            message varchar(255),
```
??

Reply:
255 is not enough. This is used for storing the error message returned by the query and will vary. In elide example we kept it same as responsebody.